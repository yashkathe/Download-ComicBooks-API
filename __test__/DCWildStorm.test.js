const { getDCComicsWildstorm } = require('../index');

test("DC WildStorm", async () => {
    expect(await getDCComicsWildstorm(1)).toMatchSnapshot();
});
