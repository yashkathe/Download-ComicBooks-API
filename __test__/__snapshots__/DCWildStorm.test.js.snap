// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DC WildStorm 1`] = `
[
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/07/Wraithborn-Wraithborn-Redux-2005-2016.jpg?fit=400%2C619&ssl=1",
    "description": "Wraithborn is a series written and drawn by Joe Benitez, about the immortal beings which inhabit Earth, preying and being preyed upon by a warrior called Wraithborn. When the Wraithborn warrior finds himself on death’s door he transfers the Wraithborn to Melanie, a timid girl young girl. She must then contend with the Immortals and others who would seek to use this power for their own purposes as well as the intended Wraithborn successor, Valin.",
    "downloadLinks": {
      "DOWNLOADNOW": "https://getcomics.info/links.php/Rs28zX1j7wWR8MhxFuqACe8Mz4UxLyH8SxZ0LdRMhFXtkB9MxdyVxveuMl9J7Us5pR1YqMWCWQmSUUFfTQViSOS4eNVtTUCo2IX0gfnJaTXoQ8Fgtr2F5EECQ2icO+dl:wXUTEFtGyddd3sLne4rfBA==",
      "MEDIAFIRE": "https://getcomics.info/links.php/uDGMBBRHDoALxdYpMkgc5YWxWub7FbsMDdWX1qPQRB3UtK9aX+skt6Omsb6vszIMsHm77rLGUl8tjs1BrVhmVIpph7UljA5oJhXqighsTxAEpe9uqf1ttESFAm843V+wJ8xVwYxiClBWfKOtjXSLdw==:BWtZXzDQ+4FLJZkXQr99bA==",
      "MEGA": "https://getcomics.info/links.php/k8tn1PRL2tJmkDq/7d/KnBl6JUsElYkeNMubwoG5DHQRxzKdasiHwMre519lH61+Fk3fYARBCT5LTnWIN1xM7FJYFYuUm9FLLwNQKdN4EUE=:idY0xz9FjhgEhd1TKf63vg==",
      "UFILE": "https://getcomics.ufile.io/r2id7sus",
      "ZIPPYSHARE": "https://www98.zippyshare.com/v/lXQWMuux/file.html",
    },
    "information": {
      "ImageFormat": "JPG",
      "Size": "209 MB",
      "Year": "2005-2016",
    },
    "title": "Wraithborn + Wraithborn Redux (2005-2016)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/05/The-Authority-Book-1-2-TPB-2020-2021.jpg?fit=400%2C615&ssl=1",
    "description": "This is the Authority. Pay allegiance, or get your head kicked in. Whether they’re up against an invasion from an alternate Earth, an international terrorist who is laying waste to major cities, or a hostile alien intelligence who calls himself God, the Authority are ready for action.",
    "downloadLinks": {},
    "information": {
      "ImageFormat": "JPG",
      "Size": "2.1 GB",
      "Year": "2020-2021",
    },
    "title": "The Authority Book 1 – 2 (TPB) (2020-2021)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2017/06/Captain-Atom-Armageddon-1-9-2005-2006-1.jpg?fit=400%2C615&ssl=1",
    "description": "Captain Atom finds himself trapped in the Wildstorm Universe. When he tries to find help, he finds that people in this world are afraid of Super-Powered-Beings. As he encounters several of the WSU characters, he begins to understand why. To make matters worse, his presence threatens to destroy the entire WSU. But there is a new host for the Void entity and she may hold the key to getting Captain Atom home and saving the WSU.",
    "downloadLinks": {
      "DOWNLOADNOW": "https://getcomics.info/links.php/Y+mh6pD5wHQtqAoNIfcSH8LSdHOBR9Wi9kmuxXVmM+PsxRRtRi23zqjHPZrmdjc2CP8wklbjUXn/AuZfEKv4xDWqqGqsBfSsSK30LeY48xBMW4OaoO2IXa+JqFPbapYG+VEne6z10DcAyWDA6z1/UaUKTryJbJtTHW2HkQmqWM0=:fOS6hEbs2gygx/gToP6y7w==",
      "MEDIAFIRE": "https://getcomics.info/links.php/thLmVydjG9vCSXfMGyNUjexF1qY+Z+eXjcWXDiDlG+CuuAD3MQQnt8/fv/uEAUaXvkeZk+2WG2BhMfl1DGt4mqyuMIMov+n8TilmVj1hVvq5D41i7dqmyYvHKy3fzjw4nLzqzWX2TLEhADooEyYVnsdV70VsM1hP1QgX0Hdo6nw=:oU9iK89ThlaP4RO9exkrGw==",
      "MEGA": "https://getcomics.info/links.php/oWzIdJL3sCRQ8w2M9rfrRztXisHMWvJryFyy5ld8AjjbQWkLTN8NmZH1I14QZO8J1Vsxx3aIEe8FwfLxudOOo+Dt9UiqQqgMq+YY/n2aqOg=:0mHUQhaRxjNcvJotsMOwAw==",
      "UFILE": "https://getcomics.ufile.io/wn7sg0ud",
      "ZIPPYSHARE": "https://www27.zippyshare.com/v/4CTo8mC2/file.html",
    },
    "information": undefined,
    "title": "Captain Atom – Armageddon #1 – 9 (2005-2006)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2021/12/Kane-and-Lynch-1-6-2010-2011.jpg?fit=400%2C614&ssl=1",
    "description": "Following the bullet-ridden finale to the best-selling video game, Kane and Lynch decide what’s best for everyone concerned is that they go their separate ways. That is, until a contract is put out on their collective lives for a huge bounty – and every assassin and two-bit goon around the world is gunning for the volatile pair. Can the unrepentant traitor and medicated psychopath work together once more to save their lives and get to the bottom of this blood-drenched conspiracy?",
    "downloadLinks": {
      "DOWNLOADNOW": "https://getcomics.info/links.php/3jG7CPA0p2Qam84Up7nN8VyzLUffcn1TVuV9o+7TmeHa+UK1Lu4JgKkJUs7aBxAv51sqneGMVdkDI/jODNSQLVAtRzTWC/KcfKU7SqDL2BxcoNwEYvXg/0XgG/vPB9pc8toZZemHQsYvcciRh543a4DdhLU8vNaxnCaq8VaodqI=:xo9deppGToMAQU1s96xAPg==",
      "MEDIAFIRE": "https://getcomics.info/links.php/mNMuMDc0v2aeN1udO15zDvsLsnw6xJLeJ2l18PZtIR28o6gENh451sMwiA/RO7fT5TlmDKrrYvUTcxf5PijyQOUeQwxwKFz97OXYV36wD6tiJVThBO9cns8Nd0Hi3e5qvnpSf21BFpkibSlmuebCEPdGHryyR2mbmvbLrEOaTBo=:VYy6RsUU5yMuMTDynE4hxw==",
      "MEGA": "https://getcomics.info/links.php/G9aaM+SsMGfv1LOIIb5urxL5voz1MYrAwv1J3P9gemQguAWV/RWKPub2qjRWTGdq7YgW2Oh3BY+goZdKiaY1fh+N5H5QRAgMHjdUZ2g1Dgk=:eLaNcWM/kK9eHm5mwx4QAw==",
      "UFILE": "https://getcomics.ufile.io/azyicr4o",
      "ZIPPYSHARE": "https://www79.zippyshare.com/v/qFD0pBZG/file.html",
    },
    "information": {
      "ImageFormat": "JPG",
      "Size": "122 MB",
      "Year": "2010-2011",
    },
    "title": "Kane and Lynch #1 – 6 (2010-2011)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2021/10/Ratchet-and-Clank-1-6-2010-2011.jpg?fit=400%2C618&ssl=1",
    "description": "With Dr. Nefarious defeated and the universe safe, galactic heroes Ratchet and Clank return to the Solana galaxy for some well-earned rest and relaxation. But when entire planets begin disappearing from the universe without a trace, Galactic President Qwark calls them back into action to solve the mystery. As the rest of the universe erupts in panic, the duo must embark on an epic adventure that will pit them against their most capable adversary to date: a Markazian planet thief named Artemis Zogg!",
    "downloadLinks": {
      "DOWNLOADNOW": "https://getcomics.info/links.php/inv0t/3WWU9ipgpxs6QiHA7tvjM+zZPuFX469H2wpGnDE9gjJnp2WMGba1TrStoRPmWnzBHEWQKQfFfVIaygneq4eckWD5mN7yOoIrSECOQYUHSHDW+r5cx4Eme17dQTJZlsX1KaAbqNERyjnqYOVPWcfxJkq9dHU88Lz408CGI=:ILXEZbCtPrBH55m1Bq6IQQ==",
      "MEDIAFIRE": "https://getcomics.info/links.php//Ng9Io8EXrmdgmikncdRUFQrSAJX5+FhhWGiAnxSUGoORvyAgaDxngGY4vBwOEOLihSZFYbjpkqHr7hxBJghTVZdIS//Xakg06F5lv1eJbcrWPGcaxtVSFyK8jtZQSAufUZ4JsF9tUIKwFClONG0td1iUNcM3hbLZUCBdxjqqYw=:SUHxkz78jnCTqQLX+cfyaA==",
      "MEGA": "https://getcomics.info/links.php/0SnuZ8T9PnvZvWKNvSTuEciLjTGAsPtEHbRObe7QoSED2MGlTDqATxi9FOzLEmExe6IWtED7P867Z9/wPSCQJXfVtDnDBtzmbykWnFosuIU=:VhVSAG66HvMZ63AFBkr7kg==",
      "UFILE": "https://getcomics.ufile.io/iq3lkz5m",
      "ZIPPYSHARE": "https://www68.zippyshare.com/v/rIYBDm8C/file.html",
    },
    "information": {
      "ImageFormat": "JPG",
      "Size": "119 MB",
      "Year": "2010-2011",
    },
    "title": "Ratchet and Clank #1 – 6 (2010-2011)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2021/08/WildCats-1-30-2008-2011.jpg?fit=400%2C615&ssl=1",
    "description": "Following the cataclysmic conclusion to NUMBER OF THE BEAST comes the dawn of a whole new age: WILDSTORM WORLD’S END! The heroes failed to save the planet from destruction and now struggle to survive in a brutal, post-apocalyptic environment…but with humanity largely dead and the Earth broken, what does the term “hero” even mean?",
    "downloadLinks": {},
    "information": {
      "ImageFormat": "JPG",
      "Size": "761 MB",
      "Year": "2008-2011",
    },
    "title": "WildCats #1 – 30 (2008-2011)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2021/07/Star-Trek-Divided-We-Fall-1-4-2001.jpg?fit=400%2C613&ssl=1",
    "description": "Things don’t look good for the Trill homeworld as a rebellion has devastated their society. A small group of terrorists are attacking joined trills and there full intention seems to be killing everyone of them. Odan a great ambassador and friend to many of the U.S.S. Enterprise E Crew has been attacked and Lenara Kahn a renowned scientist and an old lover and wife of Dax is kidnapped and possibly murdered. Unknown about Khan’s situation Ezri Dax jumps into action to save her in a ransom demand. What Ezri also doesn’t know is she is a target and the leader behind all this trouble is her old enemy. This comic takes place after the last episode of Deep Space Nine (DS9) and the Next Generation film Star Trek: Insurrection but before the Next Generation film Star Trek: Nemesis. This is determined because Benjamin Sisko is no longer on the DS9 station and Data, William Riker, and Deanna Troi are still apart of the Enterprise crew.",
    "downloadLinks": {
      "DOWNLOADNOW": "https://getcomics.info/links.php/IMH8xTXQQKjBFaK0PJz4+R1Zb4ARiun7NAfYXqHPEkEg2IqzlyB7e98QfbE9RTmYVSj7DnZjHvax2QP57RKfhj16t5bFsEX4S7As2ted5jmWFT94sSCo66jfcGmMyJeV3td9LzdzVMdAjHnef6XuSw==:5Ni8ygvSbYwlGI0/DLzifw==",
      "MEDIAFIRE": "https://getcomics.info/links.php/+G4G79F1w7MEa0dD6ibKb+LfPy8KWAXR9DXLT/1b/aTiV8ZtxU43TAXyAQ5DWsYbrRNMlPL7nKQz6sKj46ER0Ki4RKUIa/RmASG8ih+r5HYBE8Sryue4+9yjX8O+0Sok:cYr+C5R4VXq/Gc50KUqiaA==",
      "MEGA": "https://getcomics.info/links.php/mixEVXr5cJ0jfr/Jms8fXf8DTg3Yod48ji2XLVvT/N39A9ssjYOI2Y4JAI0whDvHAm1pk6syNP2JBIpUs1Q3Dsua889kfmKvqf73fclwihk=:7QuL9G0WfvfgwQBpAvHVDg==",
      "UFILE": "https://getcomics.ufile.io/fc6we44t",
      "ZIPPYSHARE": "https://www35.zippyshare.com/v/rylBLvGl/file.html",
    },
    "information": {
      "ImageFormat": "JPG",
      "Size": "41 MB",
      "Year": "2001",
    },
    "title": "Star Trek – Divided We Fall #1 – 4 (2001)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2021/07/Team-7-Dead-Reckoning-1-4-1996.jpg?fit=400%2C608&ssl=1",
    "description": "A superpowered soldier attacks Lynch telepathically. To relieve the psychic pressure, Lynch tears his own eye out, resulting in a destructive blast that kills the soldier. Cole Cash, Alex Fairchild, Stephen Callahan and Philip Chang quit Team 7. Stephan Callahan announces that he and his wife, Rebecca “Becky” Rainmaker, are expecting a child (Sarah Rainmaker).",
    "downloadLinks": {
      "DOWNLOADNOW": "https://getcomics.info/links.php/CajGtQA3Vgoi5y/xCzrIl6cgmoOhu1gPAE3fhMvkcoCcjCjDJUdhMoWJFgs92ZuXvR7s3yGRrrTmLoFdD5n8zZavmjhgQHiahIgGZyGXaZC43W99YL7/tmVQMqSI5iyhLXURrIWfam5c0yP/fvloWA==:rcAsZTkZN9AMTFvj3OD7Xw==",
      "MEDIAFIRE": "https://getcomics.info/links.php/2KuM33Tm1Jus1c+Y0RgTyonwIUa94G4BTca5/7xh2OI3ujq08bkpF9Did9pN8fQEq+Y8Y4rmuDo1vO0PjO7upCdz7tryaE+xhGykEs74CDPGNzTV5XW+z6nZ5tVj3z2e:YRD2QT7DpVdt8CGx6e900A==",
      "MEGA": "https://getcomics.info/links.php/0BfH3UTxjjXK2l0josbIq1AqYYCi7SLANZmgfOfVLJtNkHrj5d7/XKhg0Higu0RhJ8tcbxtZlVX/0ew3js07CCTyxyxzni0kFCttpbtHSvw=:+tMeRkm/rrVGydjpLAvjjA==",
      "UFILE": "https://getcomics.ufile.io/1b42v78o",
      "ZIPPYSHARE": "https://www81.zippyshare.com/v/1MRG0WMp/file.html",
    },
    "information": {
      "ImageFormat": "JPG",
      "Size": "35 MB",
      "Year": "1996",
    },
    "title": "Team 7 – Dead Reckoning #1 – 4 (1996)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2018/05/Zealot-1-3-1995-1.jpg?fit=400%2C615&ssl=1",
    "description": "Zealot in her own series! Zealot begins to tell Voodoo some of her origins and of her complicated relationship with Prometheus. Zealot recounts a tale of a run in with a former Kherubim whom she rescued from imprisonment aboard ship that brought them to earth. Promethos in turn rescues Zealot from a group of her former sisters, The Coda, and makes her an offer to rule by his side.",
    "downloadLinks": {
      "DOWNLOADNOW": "https://getcomics.info/links.php/gsviy85DXWacs0HZiLq3dygGw9gqT27nJyrHelLb1Hg/1YufrZnTl8jnPXC5nz3kjNp8PzYRSPeyQuPNeUTem78jW/RsHQ6+nTu2mLd06wI=:gGplYGGeiHHBZC4n/O7uDA==",
      "MEDIAFIRE": "https://getcomics.info/links.php/sAFNPHZMDfK4RLusQ5s44RLnIvc62rQN+2FkNhXkbMad6twQ0zBK92K9AR9MnxERmKzhT2WTfKgDI1GZBHQ3SzP9a5DEhr7F5HEnsCKvOdA6SyCliGNy41q1OQ3DpQzh:yvwT3yvwhviBXocET6fGLw==",
      "MEGA": "https://getcomics.info/links.php/kYPdDcjsMgxWifnI/nk+TapVezxKrGUh+sLK6FfeG6U/f08LRzB4XkFrOp9HMIZfViYia7WJgGCPiWE6NQjjZ8OOyafbljvaJ95Pw05bhCo=:Ef75IR2tUyZtUIdqCRceaA==",
      "UFILE": "https://getcomics.ufile.io/ussiqf51",
    },
    "information": {
      "ImageFormat": "JPG",
      "Size": "100 MB",
      "Year": "1995-1996",
    },
    "title": "Zealot #1 – 3 (1995)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2018/05/Sleeper-Season-Two-1-12-2004-2005-1.jpg?fit=400%2C615&ssl=1",
    "description": "While spymaster John Lynch slept in a coma, his protégé Holden Carver was trapped undercover. Now that his star has risen in criminal mastermind Tao’s organization, Carver is shocked by the news that Lynch is awake. Torn between new loyalties and his original mission, Carver must decide whose side he’s really on – and if they’re on his.",
    "downloadLinks": {
      "DOWNLOADNOW": "https://getcomics.info/links.php/acM1x8v9GoUrUDjE4y6/fEV0TNGowUGgHBcZB0KQ/luHucF2TchFMn9vb3R/fz0Iufrf01yAPLegRXCbqLNH1v80K0DD6MhTL1Wcob0G1CGiDWboUIg/Kz5Q+k6C+2E+XbVbvSZ1tPTCegK1vGusRqxMah1FSqpR5dEjDBMlLehHww4LekCwhu40cVdzNlgs:DqNZiQ97QNGZujHShURkzw==",
      "MEDIAFIRE": "https://getcomics.info/links.php/gS8029ediZBnDgB3P7ODQEc4o+woMp4opBHQ191OebW5QtKJumtYac+92PsqNfD596wfYohSX5ttHdLTpNPEbkAvR9k35CD490bhbk6sKnfuICC4I5lL/9j/85eSz+yNP1E1UloJU9LXtPfrD6KPS8xIXbMzgQuhcaWULLn5P88=:zivggUcyWNWfkJLTV9YI8A==",
      "MEGA": "https://getcomics.info/links.php/yLV/16I8bY3gS/yzPCNnu37pK3SUaFMDGyDGAMwZZU3bJl72YnzQhos4BvT+JPq95yja/j337LmHZPan5KHs8xcx83cwi6tYYJ10y+cUpAg=:aJLD4D8XtWO9Gr7mGdg1TA==",
      "UFILE": "https://getcomics.ufile.io/m554t91q",
      "ZIPPYSHARE": "https://www13.zippyshare.com/v/9R3p9Hzn/file.html",
    },
    "information": undefined,
    "title": "Sleeper – Season Two #1 – 12 (2004-2005)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2018/08/Sleeper-Season-One-1-12-2003-2004.jpg?fit=400%2C615&ssl=1",
    "description": "Secret Agent Holden Carver is a man on a mission — only nobody left alive seems to know it.",
    "downloadLinks": {
      "DOWNLOADNOW": "https://getcomics.info/links.php/J4OHjDgJibCuvBGDg5dmJQg20LMI3HGwAgM2lTgkP+bPq+ZTYcE81BkjKrUcS0RKEh49pKmc84JRyB/Aaz4J38yNao9RsT2OSNa90JvWCMpUarkuUhjwsBUy8+azTKL0pFSe13o6utDOLR5MSZFypg==:1MSdaUrPVckTxrTO3P+CqA==",
      "MEDIAFIRE": "https://getcomics.info/links.php/r3vyHa0qRoIp6xGkV8JZXT4nOagPCRk26wWJbeFBjoSmBnsHawUoOPOa9PCQiezRbaH7uhGqj7YrJwCz6ns9bMzoNS44h9jy2rfH5tBmfr/dcprp+wCXwDyI/BWodW+XKFuCsEWFTSaVHYh2DJetyg==:FVU7loCgSSK34cLJF6OrzA==",
      "MEGA": "https://getcomics.info/links.php/P9Kx/bF+JALyKFuby16jSsE1l4XbarwxTOMzhbCnsHjMpmxEOMPnuTJDh/gKsvue7hnI6yStSPALW1wrNW7fD5KFIUn08YXUmC2nlE8yPV0=:3YAn/fYl7EOZSQIbXLZOAg==",
      "UFILE": "https://getcomics.ufile.io/wvf30zxq",
      "ZIPPYSHARE": "https://www16.zippyshare.com/v/rAlgtK6k/file.html",
    },
    "information": undefined,
    "title": "Sleeper – Season One #1 – 12 (2003-2004)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2018/05/Point-Blank-1-5-2002-2003-1.jpg?fit=400%2C615&ssl=1",
    "description": "Who shot John Lynch?",
    "downloadLinks": {
      "DOWNLOADNOW": "https://getcomics.info/links.php/2EI0cnNiScmvjKGbLuUoKL5tNROzDfJu28ohSOldJJiydfg75CIJa5Bx4vCnfrILslGnWEyVR7G9AXdleewI2P4PrG/OrtSqZFCbYZ8i/LK5XtOqHd4U+hA9SNNcOo5V:su/zMYTb9rt6HPmuT1HeVw==",
      "MEDIAFIRE": "https://getcomics.info/links.php/NKZwwU0S8awMiuIPO8jks5/n7spqu9GWbRy+ToVkxo1hM/bjiAVycvfayHxk1ZW/fHqJQ0jkQBUtNGcU4AbgDL9D1KlW9O8BsmcWzhp+y7T8V/FdVNzlnmmW8XOVgdMd:BUxXwC1owyy2kyutatcs8g==",
      "MEGA": "https://getcomics.info/links.php/wti1DjXJWZkBvVWBpzV65k6ZlXT5K32hfVgZMt2QEm9VrWPlH+lPZyLH/FAj90HUV21WxcplwlFurnVVXBMDxkhXUnmku33nRj4NbhHxJK4=:zTk1RkKXrramcsL63Ts1zg==",
      "UFILE": "https://getcomics.ufile.io/0keg8kn9",
      "ZIPPYSHARE": "https://www16.zippyshare.com/v/Jtk1AOnE/file.html",
    },
    "information": undefined,
    "title": "Point Blank #1 – 5 (2002-2003)",
  },
]
`;
